packages/smooth_app/lib/pages/product/product_image_viewer.dart

```diff
@@ -66,6 +66,7 @@ class _ProductImageViewerState extends State<ProductImageViewer> {
           elevation: 0,
           title: Text(imageData.title),
           leading: SmoothBackButton(
+            iconColor: Colors.white,
```

Comment:
Please remove this part: it has nothing to do with the issue and I believe there's another pending issue/PR on the same subject.
Let's avoid a collision.

Reply:
It is not there in my commit